pearcej/cppds
# Add definitions for Infix, Prefix, and Postfix and index them

# Description

I added definitions for Infix, Prefix, and Postfix and also indexed them before that they weren't defined and also indexed now they are indexed also defined in the new textbook.
## Related Issue
fix #440 
Reviewed by: @njounkengdaizem 




## How Has This Been Tested?
1. Run it locally.
2. See changes.
3. Take a screenshot.
![image](https://github.com/pearcej/cppds/assets/117699634/e058fb65-9922-480f-a6f3-ab11a68d3ee3)
![image](https://github.com/pearcej/cppds/assets/117699634/c1725a39-ecd7-4e99-ae58-cbfe98d10d48)

![image](https://github.com/pearcej/cppds/assets/117699634/a52d85bf-5b5c-418f-91cf-ca636b667f7e)
